happybase/connection.py

```diff
@@ -81,6 +84,11 @@ class Connection(object):
     process as well. ``TBinaryAccelerated`` is the default protocol that
     happybase uses.
 
+    The optional `use_kerberos` argument allows you to establish a
+    secure connection to HBase. This argument requires a buffered
+    `transport` protocol. You must first authorize yourself with
+    your KDC by using kinit (e.g. kinit -kt my.keytab user@REALM)
```

Comment:
``versionadded`` comments should be added for both `use_kerberos` and `sasl_service` args